global_translations.R

```diff
@@ -38,3 +38,34 @@ complete_trans_df <- subset(final_df, untranslated_count == 0, select = -c(fuzzy
 
 # Untranslated Packages with respective languages associated
 complete_untrans_df <- subset(final_df, translated_count == 0, select = -c(fuzzy_count))
+
+# R Language Translation Team Contact details
+library(rvest)
+library(xml2)
+library(stringr)
```

Comment:
do we know where this script will be run? if it's run on server/in Docker/somehow deployed, I would recommend using `base` R regex. `stringr` brings `stringi` which in my experience is quite a heavy dependency.

OTOH, if this script will only be run manually / locally, using `stringr` will be fine :)

Reply:
NVM, I think from index.Rmd it's clear `stringr` will be around already -- no marginal dependency here :)

Reply:
The script would be run within a GitHub Actions workflow. We'll review the use of `stringr`  - I also have the same concern

Reply:
OK, best bet is to file a follow-up bug and add a TODO referencing it... don't think it's a big priority for now